pages/common/deemix.md

```diff
@@ -0,0 +1,21 @@
+# deemix
+
+> A barebone deezer downloader library built from the ashes of Deezloader Remix.
+> It can be used as a stand alone CLI app or implemented in an UI using the API.
+> More Information: <https://deemix.app>.
+
+- Download a track or playlist:
+
+`deemix {{https://www.deezer.com/us/track/108441824}}`
+
+- Download track / playlist at a specific bitrate:
+
+`deemix --bitrate {{FLAC / MP3}} {{url}}`
```

Comment:
I have tried using multiple variations of the bitrate but it always ends up ignoring them but using file types always returns the correct bitrates. i.e 320kbps for MP3 and 1411kbps for FLAC.
It might be a bug, it might be the way it was coded but since there's no proper documentation I can't say for sure.

Reply:
Ah, I see. So the filetypes are actually presets. What about this then? It would make it more consistent with other pages.
```suggestion
`deemix --bitrate {{FLAC|MP3}} {{url}}`
```